package.json

```diff
@@ -79,7 +79,7 @@
     "lint:production:stylelint": "tsc --noCheck --outDir ./dist && stylelint '**/*.styles.ts' --custom-syntax postcss-lit --no-color",
     "test": "per-env",
     "test:development": "npm-run-all --parallel test:development:web-test-runner test:development:web-test-runner:serve-coverage",
-    "test:development:playwright": "playwright test --quiet --ui --update-snapshots",
+    "test:development:playwright": "playwright test --quiet --ui-port 0 --update-snapshots",
```

Comment:
Came across this while going through the documentation. `--ui` opens Playwright's UI in a new window. `--ui-port` opens it in a new browser tab. Slightly less jarring. `0` means any available port. But we can also set it to a specific port if we want.